app/views/users/new.html.erb

```diff
@@ -0,0 +1,41 @@
+<%if @user.errors.any?%>
+  <p>The fields are mandotory</p>
+  <% @user.errors.full_messages.each do |e|%>
+    <ul style="color:red"><%= e %></ul>
+    <% end %>
+<% end%>
+
+
+<%= form_for @user do |f| %>
+  Username:</br><%= f.text_field :username, placeholder: "Your username" %></br>
+  Email:</br> <%= f.text_field :email, placeholder: "example@example.com" %></br>
+  Password:</br> <%= f.text_field :password, placeholder: "Your password" %></br>
+  </br><%= f.submit 'Submit'%></br>
+<% end %>
+
+
+
+<!--
+<form accept-charset="UTF-8" action="/users" method="post">
+  <input type="hidden" name="authenticity_token" value="<%= form_authenticity_token %>">
+  <label for="username">Username:</label>
+  <input type="text" id="username" name="user[username]"><br><br>
+  <label for="email">Email:</label>
+  <input type="text" id="email" name="user[email]"><br><br>
+  <label for="password">Password:</label>
+  <input type="text" id="password" name="user[password]"><br><br>
```

Comment:
The type of input you are using here is not the right one. There is one for passwords to avoid the input from showing the text you type and protect it.